### ISSUE TITLE
allow setting retention policy

### ISSUE THREAD

**Issue by glikson** [kislyuk/watchtower#75]
It could be useful to be able to set a retention policy on the newly created log streams.

**Comment by samamorgan** [comment]
Agreed. I was just looking into adding this myself in my override class. Would be very helpful to have this built-in.
